src/Sylius/Bundle/ReviewBundle/spec/EventListener/ReviewCreateListenerSpec.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\ReviewBundle\EventListener;
+
+use PhpSpec\ObjectBehavior;
+use Sylius\Component\Core\Model\Customer;
+use Sylius\Component\Core\Model\CustomerInterface;
+use Sylius\Component\Resource\Exception\UnexpectedTypeException;
+use Sylius\Component\Review\Model\ReviewInterface;
+use Sylius\Component\User\Context\CustomerContextInterface;
+use Symfony\Component\EventDispatcher\GenericEvent;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class ReviewCreateListenerSpec extends ObjectBehavior
+{
+    function let(CustomerContextInterface $customerContext)
+    {
+        $this->beConstructedWith($customerContext);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\ReviewBundle\EventListener\ReviewCreateListener');
+    }
+
+    function it_adds_currently_logged_customer_as_author_to_newly_created_review_if_it_has_no_author_yet(
+        $customerContext,
+        Customer $customer,
+        GenericEvent $event,
+        ReviewInterface $review
+    ) {
+        $event->getSubject()->willReturn($review)->shouldBeCalled();
+        $customerContext->getCustomer()->willReturn($customer)->shouldBeCalled();
```

Comment:
if `$review->setAuthor($customer)->shouldBeCalled()` will pass ->shouldBeCalled() is not needed.